README.md

```diff
@@ -1,20 +1,18 @@
-# Project Name
+# Smashing Magzine Heatmap
 
-> One paragraph statement about the project.
+> This project was made as a part of Microverse Curriculum for Full Stack Developer for Design-Teardown Project
 
```

Comment:
- [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.